src/channelTable/channelTablePlugin/ChannelTableRowCollection.js

```diff
@@ -23,8 +23,8 @@ define(
 
             addOrUpdateRow(row) {
                 if (this.isLADRow(row)) {
-                    this.removeRowsByObject(row.objectKeyString);
-                    super.addRows([row], 'add');
+                    this.removeExistingByKeystring(row.objectKeyString);
```

Comment:
this should go inside the `isNewerThanLAD` block in line 49

Reply:
Gocha!